ui/src/sentry.ts

```diff
@@ -0,0 +1,26 @@
+import * as Sentry from '@sentry/react'
+import { Integrations } from '@sentry/tracing'
+
+import { initSentryRoutingInstrument } from '@lib/utils/sentryHelpers'
+
+const version = process.env.REACT_APP_VERSION
+const name = process.env.REACT_APP_NAME
+const hash = process.env.REACT_APP_COMMIT_HASH
+const release = `${name}@${version}+${hash}`
+
+// this is on purpose
+console.log('current release: ', release)
+
+const SAMPLE_RATE_PROD = 0.6
+
+Sentry.init({
+  dsn:
+    'https://10930d3b0a8d427cad2147d6d845be56@o226447.ingest.sentry.io/5721090',
```

Comment:
Maybe it should be moved to env variable to be a constant :)

Reply:
Maybe there are some differences between On-premise and cloud?

Reply:
fixed.

> Maybe there are some differences between On-premise and cloud?

currently tidb-dashboard was embedded and distributed with pd, so I'd say we keep it in env variable for now and use differenct dsn when it really needs.